### ISSUE TITLE
Possible missing "toList" in algebra example

### ISSUE THREAD

**Issue by afs** [w3c/sparql-query#136]
Issue mentioned on StackOverflow

https://stackoverflow.com/questions/77978291/sparql-1-1-algebra-converting-multiset-to-sequence

**Comment by afs** [comment]
Related:

https://stackoverflow.com/questions/78011616/sparql-1-1-algebra-nesting-aggregation-in-selection

**Comment by hartig** [comment]
I agree with the proposed change in the first Stack Overflow question. That is, the input to the `Project` operator in this example must be wrapped in a `ToList` operator.

By the way, the example that this question talks about feels a bit out of place in Section 18.2.3 because, in contrast to all the other examples in this section, the example in question requires translation steps that are defined later, in Sections 18.2.4 and 18.2.5.

**Comment by hartig** [comment]
Regarding the second Stack Overflow question mentioned above, the observation in that question is right (yes, `ToList` must be applied) but  there is actually no bug in the spec. Instead, adding the `ToList` operator is a step that is required before translating the solution modifiers, as defined in [Section 18.2.5](https://www.w3.org/TR/sparql11-query/#convertSolMod).

Having said that, I think that the fact that adding the `ToList` operator is a _required_ step is not so clear in the spec.

I will create a PR that fixes both issue.
